docs/reference/cli/filters.md

```diff
@@ -83,6 +90,117 @@ This command will analyze your cluster Vulnerabilities through K8sGPT. Depending
 - Error: critical Vulnerability found ID: CVE-2019-8457 (learn more at: https://avd.aquasec.com/nvd/cve-2019-8457)
 ```
 
+## Prometheus
+
+Activate the [Prometheus](https://prometheus.io) integration:
+```bash
+k8sgpt integration activate prometheus --namespace <namespace>
+```
+
+The Prometheus integration does not deploy resources in your cluster. Instead,
+it detects a running Prometheus stack in the provided namespace.
+
+If successful, you should see the following success message displayed:
+```
+Activating prometheus integration...
+Found existing installation
+Activated integration prometheus
+```
+
+Otherwise, it will report an error:
+```
+Activating prometheus integration...
+Prometheus installation not found in namespace: <namespace>.
+                Please ensure Prometheus is deployed to analyze.
+Error: no prometheus installation found
```

Comment:
I like that you included the error alternative

Reply:
Thanks!